src/containers/CreateAdditionalOrganization.jsx

```diff
@@ -0,0 +1,169 @@
+import PropTypes from "prop-types";
```

Comment:
In most of the codebase, we would make this a component and then create a dialog in the AdminOrganizationDashboard.  I suppose this makes it possible to get a link directly to adding an organization (e.g. on first startup)?

Reply:
Do you mean a [material UI dialog](https://material-ui.com/components/dialogs/)? 
Yeah I think we kinda just added this to `routes` as a first step as we were getting things set up, so I don't think any of us are tied to this being a route.
I'd be happy to change if you or someone else thinks it's worth it
cc @ibrand  @lesia @ali @kelwen